src/index.js

```diff
@@ -1,41 +1,40 @@
+const environment = require("./handlers/environment-handler");
+const AWS = require('aws-sdk')
+const lambda = new AWS.Lambda()
+const deployEnv = environment.environment;
+const querystring = require("querystring");
+
 exports.slack_commands_sync = async (event, context) => {
-    if (event === null || event === undefined || event.body === undefined) {
+    if (event["detail-type"] === "Scheduled Event") {
```

Comment:
The new way that the lambdas are being invoked results in an different event payload from the EventBridge rule that's keeping the lambdas hot